manager/integration/tests/test_node.py

```diff
@@ -320,25 +320,34 @@ def test_replica_scheduler_too_large_volume_fit_any_disks(client):  # NOQA
         node.diskUpdate(disks=update_disks)
 
     # volume is too large to fill into any disks
+    volume_size = 4 * Gi
     vol_name = common.generate_volume_name()
-    volume = client.create_volume(name=vol_name,
-                                  size=str(4*Gi), numberOfReplicas=len(nodes))
+    client.create_volume(name=vol_name, size=str(volume_size),
+                         numberOfReplicas=len(nodes))
     volume = common.wait_for_volume_condition_scheduled(client, vol_name,
                                                         "status",
                                                         CONDITION_STATUS_FALSE)
 
-    # reduce StorageReserved of each default disk
+    # Reduce StorageReserved of each default disk so that each node can fit
+    # only one replica.
+    # If one node can fit multiple replicas and the following node update gets
+    # stuck for seconds,
+    # there may be multiple replicas scheduled in the same node.
+    # To prevent this race condition in the test, `storageReserved` should be
+    # set with a appropriate value so that one node can fit one replicas only.
+    schedulable = int(volume_size * 1.5 / 5)
```

Comment:
`5` is the value of OverProvisioningPercentage.

Also, name the value `neededForScheduling` instead.